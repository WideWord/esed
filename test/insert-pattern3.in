fgfdg
433
dfdfdds
433
dsssfs
efesfethg
efds4tfd
433dfrtrge