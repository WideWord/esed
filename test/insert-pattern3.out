fgfdg
xoxo
433
dfdfdds
xoxo
433
dsssfs
efesfethg
efds4tfd
433dfrtrge